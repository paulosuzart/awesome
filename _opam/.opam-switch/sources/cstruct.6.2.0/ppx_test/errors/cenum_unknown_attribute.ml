[%%cenum
  type t =
    | A
  [@@uint9_t]
]
