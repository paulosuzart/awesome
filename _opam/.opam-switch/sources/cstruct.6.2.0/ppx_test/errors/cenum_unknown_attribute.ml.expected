Error: ppx_cstruct: enum: unknown width specifier uint9_t
